src/fastnetmon.conf

```diff
@@ -165,6 +165,9 @@ sflow_host = 0.0.0.0
 # This option is not default and you need build it additionally
 # sflow_lua_hooks_path = /usr/src/fastnetmon/src/sflow_hooks.lua
 
+# sFLOW ethertype of outer tag in QinQ
```

Comment:
Small nit, please replace to "sFlow".  That's how authors (https://sflow.org/) write it.

Reply:
Fixed